charts/zeebe-benchmark/Chart.yaml

```diff
@@ -9,8 +9,8 @@ sources:
   - https://github.com/camunda/camunda
 dependencies:
   - name: camunda-platform
-    repository: "https://helm.camunda.io"
-    version: 10.2.1
+    repository: "oci://ghcr.io/camunda/helm"
+    version: "0.0.0-8.6.0-alpha4"
```

Comment:
:question: Will we be able to somehow auto update this? :S Or do we need to update manually on every release? I mean, it's the same as now, I'm just wondering for the future as this version looks so weird :smile:

Reply:
I'm also not sure yet. Not sure how good renovate is to auto update such. We will see. Hopefully I can migrate to the other chart again soon.